app/Http/Controllers/PreferenceController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Facades\Preferences;
+use Illuminate\Http\Request;
+
+class PreferenceController extends Controller
+{
+    public function index()
+    {
+        return view('preferences', [
+            'currentResourceLanguagePreference' => Preferences::get('resource-language-preference'),
+            'resourceLanguagePreferences' => Preferences::preferences()['resource-language-preference']['options'],
```

Comment:
That may be a good place to go eventually. Right now, the exact shape of this preferences object is under a bit of flux. 

One reason the thing you've described here might not work is because preferences can describe at *least* two things: First, the actual user's preferences, and second, the definition of places a user can have a preferences.

So, if I saw `Preferences::get('resource-language')` I would assume that I'm asking for the user's preferences.

Yet, here, we're actually asking about the *potential* preferences. 

I agree the current syntax is gross, but I'm not 100% sure what the better syntax is.